charts/ai-software-templates/chatbot/templates/application-gitops-job.yaml

```
@@ -12,7 +12,78 @@ spec:
     spec:
       containers:
         - name: repo-creator
-          image: quay.io/redhat-ai-dev/helm-chart-application-gitops:latest
+          command:
+            - /bin/bash
+            - -c
+            - |
+              #!/usr/bin/env bash
+              cd /tmp || exit
+              wget https://github.com/cli/cli/releases/download/v2.62.0/gh_2.62.0_linux_amd64.tar.gz
+              gunzip gh_2.62.0_linux_amd64.tar.gz
+              tar xf gh_2.62.0_linux_amd64.tar
+              export PATH=./gh_2.62.0_linux_amd64/bin:$PATH
```

I think it would be worth including the full path here, as I `think` that if you cd to another directory for some reason, that relative path is not going to work anymore.
```suggestion
              export PATH=$(pwd)/gh_2.62.0_linux_amd64/bin:$PATH
```

---

yep